backend/src/main/java/com/team4/backend/configuration/SecurityConfiguration.java

```diff
@@ -45,7 +45,8 @@ public SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http) {
                 .pathMatchers(HttpMethod.OPTIONS).permitAll()
                 .pathMatchers("/user/login").permitAll()
                 .anyExchange().authenticated()
-                .and().build();
+                .and().cors().and().csrf().disable()
```

Comment:
Please add a TODO comment about using @Profile to disable CORS and CSRF only when developing

Reply:
CORS and CSRF are required to be able to access the services from an independent web app (from an other domain).